shuup/xtheme/_theme.py

```diff
@@ -161,6 +166,72 @@ def get_view(self, view_name):
         """
         return None
 
+    def get_all_plugin_choices(self, empty_label=None):
+        """
+        Get grouped list of 2-tuples (identifier and name) of all available Xtheme plugins.
+
+        Super handy for `<select>` boxen.
+
+        :param empty_label: Label for the "empty" choice. If falsy, no empty choice is prepended
+        :type empty_label: str|None
+        :return: List of 2-tuples
+        :rtype: Iterable[tuple[str, str]]
+        """
+        choices = []
+        if empty_label:
+            choices.append(("", empty_label))
+        choices += [(_("Global plugins"), self._get_global_plugin_choices())]
+        choices += [(_("Theme plugins"), self._get_theme_plugin_choices())]
+        return choices
+
+    def _get_theme_plugin_choices(self, empty_label=None):
+        """
+        Get a sorted list of 2-tuples (identifier and name) of available theme specific Xtheme plugins.
+
+        Handy for `<select>` boxen.
+
+        :param empty_label: Label for the "empty" choice. If falsy, no empty choice is prepended
+        :type empty_label: str|None
+        :return: List of 2-tuples
+        :rtype: Iterable[tuple[str, str]]
+        """
+        choices = []
+        if empty_label:
```

Comment:
Ok. Maybe there is a usecase to just get the theme plugin choices. I guess the `_get_global_plugin_choices` shouldn't be private either.